django_telegrambot/apps.py

```diff
@@ -113,19 +120,11 @@ def getBot(cls, bot_id=None, safe=True):
 
     @classmethod
     def get_updater(cls, bot_id=None, safe=True):
-        if bot_id is None:
-            return cls.updaters[0]
+        bot = cls.getBotById(bot_id, safe)
```

Comment:
get_updater would always return None when no bot_id is passed, which makes bot_id more or less required to use this funktionallity. Is there any purpose on making bot_id an optional keyword argument or a case where it could be use full to return None? 

However the previous behavior returning the first updater, if the bot_id is not valid seems worse..

Reply:
Well I wanted to return the first bot available; it’s a bug. I guess this behavior of returning the first bot isn’t bad though, many people only have one bot in their site.

Reply:
No, the getBotById will always return the first bot available if the bot_id=None. the `return None` is for times when the bot_id isn't `None`, but also it isn't a valid bot_id found in settings.

Reply:
Even if this behavior is not ok, we can't change it because people are using it like get_dispatcher() to use their only and single bot.